server/models/handlers/availability.js

```diff
@@ -1,29 +1,24 @@
 // Gets the time availabilities for a specific worker for a specific school.
-async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
-        {
-            workerTimeslotId: 1,
-            startTime: '08:00:00', 
-            endTime: '08:30:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 2,
-            startTime: '08:30:00', 
-            endTime: '09:00:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 3,
-            startTime: '09:00:00', 
-            endTime: '09:30:00',
-            date: '2020-10-20'
-        }
-    ];
+const _ = require('lodash');
+const availabilityHandler = require('../db/availability');
+//need userTypes or Workerstatus?
+const UserTypes = require('../../constants/userTypes.json');
+
+//
+async function getAvailabilityDetails(workerId, schoolId, startTime, endTime)  {
+   
+    // Don't need this becuase the db will return an error if the worker credentials are invalid.Searches the worker table to see if it is valid worker credentials
```

Comment:
Hi Nancy, 
I am a little confused about this comment and its context here. What exactly is not needed? Is it the following line? If so, I would suggest removing line 11 and then revising the comment to be `// Worker credentials do not need to be checked as the getAvailabilityDetails method will return an error if ...` to prevent misunderstandings in the future.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've deleted the code which checks if the worker credentials are valid. This is because the getAvailabilityDetails method will return an error if the worker credentials are invalid.
You can now see these changes in line 8 of server/models/handlers/availability.js.

Please confirm if this takes care of your suggestion!